python/ray/tune/trial.py

```diff
@@ -497,6 +516,14 @@ def update_last_result(self, result, terminate=False):
         self.last_update_time = time.time()
         self.result_logger.on_result(self.last_result)
 
+        try:
+            if not math.isnan(result["episode_reward_mean"]):
```

Comment:
Can we make this configurable instead? e.g.

If I want to keep the last 5 checkpoints:
`--keep-checkpoints=5`  # checkpoint keep ranking defaults to timestamp or something

If I want to keep the last 5 checkpoints ranked by reward:
`--keep-checkpoints=5 --checkpoint-score-attr=episode_reward_mean`

If I want to keep the last 5 checkpoints ranked by loss:
`--keep-checkpoints=5 --checkpoint-score-attr=-validation_loss`

Reply:
Totally agree, will update.